test/common/access_log/access_log_formatter_test.cc

```diff
@@ -1563,6 +1590,7 @@ TEST(AccessLogFormatterTets, JsonFormatterTypedFilterStateTest) {
 
   std::string json =
       formatter.format(request_headers, response_headers, response_trailers, stream_info);
+  std::cout << json << std::endl;
```

Comment:
?

Reply:
A forgotten debug output, it will be cleaned up.